src/dotnet/Fable.Client.Browser/Fable.Client.Browser.fsproj

```diff
@@ -30,8 +28,7 @@
     <Compile Include="../Fable.Compiler/FSharp2Fable.Util.fs"/>
     <Compile Include="../Fable.Compiler/FSharp2Fable.fs"/>
     <Compile Include="../Fable.Compiler/Fable2Babel.fs"/>
-
-    <Compile Include="../Fable.Tools/State.fs"/>
+    <Compile Include="../Fable.Compiler/State.fs"/>
```

Comment:
Ok, thanks a lot for the clarification. We can check that later and merge this now :+1: